include/matx/operators/svd.h

```diff
@@ -191,7 +192,7 @@ namespace detail {
       using svd_xform_op = bool;
 
       __MATX_INLINE__ std::string str() const { return "svdpi(" + get_type_str(a_) + ")"; }
-      __MATX_INLINE__ SVDBPIOp(const OpA &a, int iterations) : a_(a), iterations_(iterations)
+      __MATX_INLINE__ SVDBPIOp(const OpA &a, int max_iters=10, float tol = 0.0f) : a_(a), max_iters_(max_iters), tol_(tol)
```

Comment:
I think the default should only be in the user-facing interface since it would get carried down to here

Reply:
fixed